Orange/widgets/evaluate/tests/test_owtestlearners.py

```diff
@@ -247,6 +247,38 @@ class NewRegressionScore(RegressionScore):
             del Score.registry["NewClassificationScore"]
             del Score.registry["NewRegressionScore"]
 
+    def test_target_changing(self):
+        data = Table("iris")
+        w = self.widget  #: OWTestLearners
+
+        w.n_folds = 2
+        self.send_signal(self.widget.Inputs.train_data, data)
+        self.send_signal(self.widget.Inputs.learner,
+                         LogisticRegressionLearner(), 0, wait=5000)
+
+        average_auc = float(w.view.model().item(0, 1).text())
+
+        w.class_selection = "Iris-setosa"
+        w._on_target_class_changed()
+        setosa_auc = float(w.view.model().item(0, 1).text())
+
+        w.class_selection = "Iris-versicolor"
+        w._on_target_class_changed()
+        versicolor_auc = float(w.view.model().item(0, 1).text())
+
+        w.class_selection = "Iris-virginica"
+        w._on_target_class_changed()
+        virginica_auc = float(w.view.model().item(0, 1).text())
+
+        self.assertIsNotNone(average_auc)
+        self.assertIsNotNone(versicolor_auc)
```

Comment:
Why is checking that the aucs are not None necessary?
If they would not be computed, the conversion to float would fail before this anyway.